testing/src/cypress/e2e/02-institution/01-login-institution-page.cy.ts

```diff
@@ -1,22 +1,34 @@
 import DashboardInstitutionObject from "../../page-objects/Institution-objects/DashboardInstitutionObject";
 import InstitutionCustomCommand from "../../custom-command/institution/InstitutionCustomCommand";
+import LoginInstitutionObject from "../../page-objects/Institution-objects/LoginInstitutionObject";
 
-describe("Login Page", () => {
-  const dashboardInstitutionObject = new DashboardInstitutionObject();
-  const institutionCustomCommand = new InstitutionCustomCommand();
+const dashboardObject = new DashboardInstitutionObject();
+const iCC = new InstitutionCustomCommand();
+const loginObject = new LoginInstitutionObject();
 
-  const url = Cypress.env("institutionURL");
+const LOGIN_URL = Cypress.env("TEST").BASE_URL + "/institution/login";
+const UNAME = Cypress.env("TEST").UNAME_1;
```

Comment:
What is purpose of suffixing the the username and password variable(UNAME_1 and PASS_1)

Reply:
Since we have more than one test users in our lower environment, wanted to keep them unique/incremental

Reply:
Addressed